### PR TITLE
Datapage UX Update

### PR DESCRIPTION
**Pull request by SumitMish23**
**---------> UI & UX Changes For DATA-BOARD <---------**

 **NavBar**
-> Changed the Background Colors & Box Shadows for the Docs,Export and Theme Buttons.
-> Updated the CSV and Theme Buttons in export-to-csv and graph-data components.
-> Updated the Size of Graphlooker Logo Icons for screen sizes below 800px.
-> Added Border-Shadow in the navbar as demonstrated in the Figma.

**AppBar (Drawer)**
-> Added App Heading class which consists of **Heading title** and **Drawer Collapse button** for closing the app bar.
-> Changed the text fonts & size of the List Item inside the ListItem Component.
-> Replaced the existing logo for **displaying** the entities.
-> Changed the Margins & Padding around each list item.

**Graph-Data**
-> Added **Copy** Functionality ,So the user can copy any given data in the table cell by clicking on it.
-> Added CSS For Copy Icon.
->Removed **Tooltip** from the table cell.

**Graph-Data-Table**
-> Replaced the existing **ternany operators** for selecting a classname with the **specified function** named as getClassNameByEntity.
-> Fixed the issue of rolling pagination container by adding the classes in CSS.
-> Changed the Button and Icons for **pagination container**.
-> Added CSS for Dynamically changing **data-table height** based on the filters selected.
->Added Customized **Scrollbars** for the graph data table.

**Additional Changes**
->Added Background and Border Shadows in the theme variable file.
->Added Text Name in the Constant File. 
->Added Fonts('Inter') in the index file.
->Added SVG Icons for the Buttons.
 
**Home**
       (Navbar & Footer)
->Updated the Navbar Icons & Height of home navbar in accordance with data board component.
->Changed the Footer into Theme Based Div.

**-------> Images <-------**
Before the changes have implemented:
image

After the Changes :
(i) **FULL PAGE**
image
image

(ii)**Iphone SE**
image

(iii)**Iphone XR**
image

(iv)**Pixel 5**
image

(v)**Ipad Mini**
image

(vi)**Samsung Galaxy** 
image
image

### CODE REVIEW

File: src/components/GraphDataTable/graph-data-table.scss [hunk 1]
```diff
@@ -1,41 +1,59 @@
 @import '../../utility/theme/variable.scss';
+
 .FilterData {
   max-width: calc(100vw - 310px);
   overflow: auto;
   padding: $size3;
   margin: $size3;
 }
+
```
Comment (harshrjjpt) [inline src/components/GraphDataTable/graph-data-table.scss:9]: remove empty lines..

File: src/utility/theme/dark-theme.scss [hunk 2]
```diff
@@ -22,5 +22,6 @@
   --listIconColour: white;
   --loaderBackground: #000;
   --loaderTextColor: #fff;
-  --iconBackgroundColor: #fff;
+  --iconBackgroundColor: #44ABE7;   //2.0
```
Comment (harshrjjpt) [inline src/utility/theme/dark-theme.scss:25]: remove comment

File: src/components/GraphData/graph-data.tsx [hunk 3]
```diff
@@ -195,11 +202,7 @@ const GraphData: React.FunctionComponent<RouteComponentProps<any>> = ({ location
         <AppBar position="fixed" className="app-bar">
```
Comment (harshrjjpt) [inline src/components/GraphData/graph-data.tsx:202]: graphlooker logo width is not perfect below 600px.

File: src/components/PrimaryMenu/primary-menu.scss [hunk 4]
```diff
@@ -30,6 +32,7 @@
   border-radius: 5px;
   padding-top: $size8;
   padding-bottom: $size8;
+  background-color:#ECEAEA;
```
Comment (tech-nerd) [inline src/components/PrimaryMenu/primary-menu.scss:35]: get it from varibale

File: src/components/PrimaryMenu/primary-menu.scss [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
   padding: 0;
   display: flex;
   margin-left: $size6;
+  margin-right: $size8;
+  border-radius: 5px;
 
   .dropdown-arrow {
-    color: #03000c;
+    height: 18px;
+    color: #FFFFFF;
```
Comment (tech-nerd) [inline src/components/PrimaryMenu/primary-menu.scss:18]: get it from varibale

**Review by tech-nerd — CHANGES_REQUESTED** [review]
image
- In mobile download, icon position is not correct or in center

image

- In mobile and web view pagination buttons are not correctly aligned as per UX designs

File: src/components/GraphData/graph-data.scss [hunk 6]
```diff
@@ -1,4 +1,5 @@
 @import '../../utility/theme/variable.scss';
+@import '../../utility/theme/constant-color-variables.scss';
```
Comment (tech-nerd) [inline src/components/GraphData/graph-data.scss:2]: rename "constant-color-variables.scss" to "common.scss"

File: src/components/GraphDataTable/graph-data-table.scss [hunk 7]
```diff
@@ -58,25 +76,36 @@
...
         font-weight: bold;
         width: max-content;
-        &:hover {
-          border-color: $colorBlack;
+        font-style: normal;
+        font-weight: $fontWeight700;
+        font-size: 14px;
+        line-height: 17px;
+        letter-spacing: 0.1em;
+        color: #000000;
```
Comment (tech-nerd) [inline src/components/GraphDataTable/graph-data-table.scss:96]: get it from varaible

File: src/components/GraphDataTable/graph-data-table.scss [hunk 8]
```diff
@@ -94,35 +123,128 @@
...
   }
 }
+
+.tabledata-row {
+  #tableCell {
+    position: relative;
+    padding: 15px;
+
+    svg {
+      color: #000000;
```
Comment (tech-nerd) [inline src/components/GraphDataTable/graph-data-table.scss:169]: get it from variable

File: src/components/Navbar/navbar.scss [hunk 9]
```diff
@@ -158,13 +146,11 @@
...
-        margin-top: -8px;
-        width: 30px;
-        height: 30px;
-        border-radius: 8px;
+        padding: 5px;
+        width: 21px;
         svg {
-          width: 13px;
-          height: 13px;
+          margin-left: -1px;
```
Comment (tech-nerd) [inline src/components/Navbar/navbar.scss:152]: don't use negative

File: src/utility/theme/constant-color-variables.scss [hunk 10]
```diff
@@ -0,0 +1,8 @@
+//------------> Colors <-------------------
+$colorWhite:#FFFFFF;
+$sortButtonBackground:#2DA0E3;
+$filterButtonBackground:#ECEAEA;
+$collapseIconColor:#646464;
+$themeIconSvg:#040505;
+$drawerIconBackground:#dbdbdb;
+$iconbackgroundColor:#fff;
```
Comment (tech-nerd) [inline src/utility/theme/constant-color-variables.scss:8]: its same as #ffffff

**Review by tech-nerd — CHANGES_REQUESTED** [review]

**Review by tech-nerd — APPROVED** [review]

**Pull request merged**